strongback/src/org/strongback/hardware/Hardware.java

```diff
@@ -781,7 +781,7 @@ public static InputDevice driverStationJoystick(int port) {
          * @param port the port on the driver station that the flight stick is plugged into
          * @return the input device; never null
          */
-        public static FlightStick logitechAttack3D(int port) {
+        public static FlightStick logitechAttack3(int port) {
```

Comment:
Changing the name of this method will break existing robot code that uses it via an earlier version of Strongback. If you really think changing the name is important, then to keep backward compatibility we need to _add_ a method with the old name/signature and with a `@Deprecated` annotation and JavaDoc tag.